pytestqt/plugin.py

```diff
@@ -694,6 +696,25 @@ class _QtMessageCapture(object):
     def __init__(self, ignore_regexes):
         self._records = []
         self._ignore_regexes = ignore_regexes or []
+        self._previous_handler = None
+
+    def _start(self):
+        self._previous_handler = qInstallMsgHandler(self._handle)
+
+    def _stop(self):
+        qInstallMsgHandler(self._previous_handler)
+
+    @contextmanager
+    def disabled(self):
```

Comment:
Perhaps call this `disable` instead of `disabled`? Doing `qtlog.disable()` seems more natural, `disabled` sounds more like an attribute to check whether something is currently disabled.